### DISCUSSION TITLE
`zoneinfo` is not correctly sorted?

### DISCUSSION THREAD

**Discussion by jmg-duarte** [astral-sh/ruff#10846] [Q&A] [answered]
I'm opening this as a discussion because I am not sure.

Correct me if I am wrong but the import sorting does it like:
1. "Native" Python imports
2. 3rd party dependencies
3. Project modules

If I am correct, `ruff` is placing `zoneinfo` in the second group and this could be turned into an issue.
I'd be open to fix it too!

**Comment by charliermarsh** [comment, marked as answer]
What's your minimum-supported Python version? See: https://github.com/astral-sh/ruff/issues/6475.

**Reply by jmg-duarte** [reply]
I had none! `target-version = "py312"` solved it. 

Thanks and sorry for the duplicate, I hadn't found that one because I was formatting rather than checking.

**Reply by charliermarsh** [reply]
No prob at all!
